flang/test/Driver/Inputs/main_dupes.c

```diff
@@ -0,0 +1,8 @@
+#include <stdio.h>
+
+int main(int argc, char * argv[]) {
+    // Irrelevant what to do in here.
+    // Test is supposed to fail at link time.
+    printf("Hello from C [%s]\n", __FUNCTION__);
```

Comment:
Note that if you remove this line and `#include <stdio.h>` then this translation becomes super trivial, yet sufficient.

Reply:
This is also gone with the move to an LL file.